content/applications/services/helpdesk/overview/help_center.rst

```diff
@@ -0,0 +1,429 @@
+
+.. warning::
+   Since all of the *Help Center* features require integration with other applications, enabling any
+   of them may result in the installation of additional modules (or applications).
+
+   Installing a new application on a *One-App-Free* database will trigger a 15-day trial. At the end
+   of the trial, if a paid subscription has not been added to the database, it will no longer be
+   active or accessible.
+
+Knowledge
+=========
+
+Odoo's *Knowledge* application is a collaborative library where users can store, edit, and share
+information. The *Knowledge* app is represented throughout the database by a *book* icon.
+
+.. image:: help_center/help-center-knowledge-book-icon.png
+   :align: center
+   :alt: View of a message in Helpdesk focusing on the Knowledge book icon.
+
+Enable knowledge on a Helpdesk team
+-----------------------------------
+
+To enable the *Knowledge* feature on a *Helpdesk* team, go to :menuselection:`Helpdesk -->
+Configuration --> Teams` and select a team, or create a :doc:`new one <getting_started>`.
+
+When a team has been selected (or created), Odoo displays that team's detail form.
+
+On the team's detail form, scroll down to the :guilabel:`Self-Service` section. Click the box next
+to :guilabel:`Knowledge` to activate the *Knowledge* feature. When clicked, a new field labeled
+:guilabel:`Article` appears.
+
+Clicking the :guilabel:`Article` field reveals a drop-down menu. At first, there is only one option
+in the drop-down menu titled *Help*, which Odoo provides by default. Select *Help* from the
+drop-down to choose this article.
+
+.. tip::
+   To create a new article, go to :menuselection:`Main Dashboard --> Knowledge`. In the *Knowledge*
+   app, hover the cursor next to the :guilabel:`Workspace` section header, located in the left
+   sidebar. Moving the cursor there reveals a hidden *plus sign* icon (:guilabel:`+`).
+
+   Click the "plus sign" icon (:guilabel:`+`) to create a new article in the :guilabel:`Workspace`.
+   In the upper right corner of the page, click the :guilabel:`Share` button, and slide the
+   :guilabel:`Share to Web` toggle switch until it reads :guilabel:`Article Published`. It can then
+   be added to a *Helpdesk* team.
+
+Once an article has been created and assigned to a *Helpdesk* team, content can be added and
+organized through the *Knowledge* app.
+
+.. seealso::
+   - :doc:`Editing Knowledge articles </applications/productivity/knowledge/articles_editing>`
+
+Search articles from a Helpdesk ticket
+--------------------------------------
+
+When members of a *Helpdesk* team are trying to solve a ticket, they can search through the content
+in the *Knowledge* app for more information on the issue.
+
+To search *Knowledge* articles, open a :guilabel:`ticket` - either from the *Helpdesk* app
+dashboard, or by going to :menuselection:`Helpdesk app --> Tickets --> All Tickets`, and selecting a
+:guilabel:`Ticket` from the list.
```

Comment:
no gui on line 79
em (—) dash instead of hyphen

```suggestion
To search *Knowledge* articles, open a ticket — either from the *Helpdesk* app
dashboard, or by going to :menuselection:`Helpdesk app --> Tickets --> All Tickets`, and selecting a
:guilabel:`Ticket` from the list.
```